contracts/lib/LibPrices.sol

```diff
@@ -36,14 +36,19 @@ library Prices {
         pure
         returns (int256)
     {
-        return int256((averageTracerPrice - averageOraclePrice) / 90);
+        return
+            (averageTracerPrice.toInt256() - averageOraclePrice.toInt256()) /
+            90;
     }
 
     function averagePrice(PriceInstant memory price)
         public
         pure
         returns (uint256)
     {
+        // todo double check safety of this.
+        // average price == 0 is not neccesarily the
+        // same as no trades in average
         if (price.trades == 0) {
             return 0;
         }
```

Comment:
Yeah, we need to make sure that having the contract consider averagePrice to be 0 on no trades is safe. Could this mess up the funding rate towards the start of a market's deployment?

Reply:
@BenjaminPatch I'll have a look into this now. Are you happy for me to do this as part of this PR if there are changes to be made?

Reply:
Yeah I am